packages/shared/src/store/sync.ts

```diff
@@ -268,6 +269,44 @@ export const syncSystemContacts = async (ctx?: SyncCtx) => {
   }
 };
 
+export const syncContactDiscovery = async (ctx?: SyncCtx) => {
```

Comment:
do we want to confirm they have an attested phone number before trying discovery? If we know we're going to get no matches might make sense to short circuit + avoid rate limit hit